src/common/dump.c

```diff
@@ -4,6 +4,9 @@
 #include "dump.h"
 #include <stdio.h>
 #include "w25x.h"
+#include "w25x_communication.h"
+#define F_CPU SystemCoreClock
+#include "../../lib/Marlin/Marlin/src/HAL/shared/Delay.h"
```

Comment:
I don't like having dependency on Marlin in dump.c, that's completely unrelated. Don't we have a DELAY_NS in some other form? AFAIK it all boils down to tick_ns() or a similar call, which is our code.

Reply:
To my knowledge we don't have any delay function working with disabled interrupts in our codebase.

Reply:
moved to https://github.com/prusa3d/Prusa-Firmware-Buddy/pull/2042